tasks/affective_task/server_affective_task.py

```diff
@@ -1,4 +1,5 @@
 import csv
+from email import header
```

Comment:
Why do you need this?

Reply:
I'm not sure how this came in here. I was doing another project and must have accidentally put it over here. Anyway removed it.